components/raftstore/src/store/worker/pd.rs

```diff
@@ -75,6 +75,19 @@ where
     }
 }
 
+pub struct HeartbeatTask {
+    pub term: u64,
+    pub region: metapb::Region,
+    pub peer: metapb::Peer,
+    pub down_peers: Vec<pdpb::PeerStats>,
+    pub pending_peers: Vec<metapb::Peer>,
+    pub written_bytes: u64,
+    pub written_keys: u64,
+    pub approximate_size: Option<u64>,
+    pub approximate_keys: Option<u64>,
```

Comment:
Please change it to `u64` instead of `Option<u64>`

Reply:
Done.